js/faq.js

```diff
@@ -1,8 +1,34 @@
 const faqLinks = document.querySelectorAll('.faq_link');
 const currentLocation = window.location.hash;
 
+window.addEventListener('keydown', (e) => {
+  const ancTag = document.querySelector(
+    `a[href="${e?.target?.attributes?.href?.value}"]`,
+  );
+  if (e.key === ' ' && ancTag) {
+    e.preventDefault();
+    ancTag?.click();
+  }
+});
+
+const removeForOthers = (target) => {
+  faqLinks.forEach((faqLink) => {
+    if (faqLink.getAttribute('href') != target.getAttribute('href')) {
+      const faqText = faqLink.nextElementSibling;
```

Comment:
Could this be renamed to `siblingElement` or something more appropriate name? As `faqText`signifies it as a text value rather than an element node.

Reply:
changed